cmd/sync.go

```diff
@@ -44,19 +45,26 @@ var syncCmd = &cobra.Command{
     #   archs: [x86_64]
   `,
 	Run: func(cmd *cobra.Command, args []string) {
+		var ErrorFlag bool = false
```

Comment:
Nitpick: local variables are lowercase in Go - CamelCase is reserved to externally-visible symbols.